session_0/word_processing.rb

```diff
@@ -30,15 +34,30 @@ def words_with_prefix(prefix, words)
 #   the prefix
 # - similarity of (applesauce, apple) is 5 as 'apple' is the largest
 #   common prefix.
-# 
+#
 # The function `similarity_score` takes two words and returns the
 # similarity score (an integer).
 def similarity_score(word_1, word_2)
-  raise NotImplementedError # TODO
+  current_max = 0
+  current_max += 1 while word_1[current_max] == word_2[current_max] && current_max != word_1.size
```

Comment:
See if you can use `find` function here: https://apidock.com/ruby/Enumerable/find